Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -30,17 +43,12 @@ final class PledgeCTAContainerView: UIView {
     _ = ([self.youreABackerLabel, self.amountOrRewardLabel], self.amountAndRewardTitleStackView)
       |> ksr_addArrangedSubviewsToStackView()
 
-    _ = ([self.amountAndRewardTitleStackView, self.pledgeCTAbutton], self.rootStackView)
+    _ = ([self.amountAndRewardTitleStackView, self.spacer, self.pledgeButton], self.rootStackView)
```

Comment:
We had to add a spacer that would properly align the button to the right and fill in the space between the labels and the button.